stdlib/LinearAlgebra/test/adjtrans.jl

```diff
@@ -476,6 +476,16 @@ end
     @test adjoint!(b, a) === b
 end
 
+@testset "copyto! uses adjoint!/transpose!" begin
+    for T in (Float64, ComplexF64), f in (transpose, adjoint), sz in ((5,4), (5,))
+        S = rand(T, sz)
+        adjS = f(S)
+        A = similar(S')
+        copyto!(A, adjS)
+        @test A == adjS
```

Comment:
I think we need to test the alternative branch, perhaps

```suggestion
        @test A == adjS
        B = similar(S)
        copyto!(B, adjS)
        @test B[:] == adjS[:]
```
?

Reply:
The other branch is already tested in `test/copy.jl`, but if you like, we may add these tests as well.

Reply:
I see. No, then that's fine. Let's not stress CI unnecessarily.